spec/support/factories.rb

```diff
@@ -6,6 +6,13 @@
     password "password"
   end
 
+  factory :admin, class: User do
```

Comment:
Wouldn't that mean that while we'd be saving code here, we'd have to put in extra repeated code every time we want to create an admin? Instead of just calling FactoryGirl.create(:admin), we'd have to use FactoryGirl to create a user, then change the role, then save the user - every time we want to make an admin. Doesn't using FactoryGirl help us dry up that part of our code?